mmv1/products/spanner/terraform.yaml

```diff
@@ -31,6 +31,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         On older versions, it is strongly recommended to set `lifecycle { prevent_destroy = true }`
         on databases in order to prevent accidental data loss. See [Terraform docs](https://www.terraform.io/docs/configuration/resources.html#prevent_destroy)
         for more information on lifecycle parameters.
+        
+        Note: Databases that are created with POSTGRESQL dialect do not support 
```

Comment:
This is gonna be a pretty bad experience for Terraform users as it's a strong expectation in the ecosystem that a single `terraform apply` will finish setting up your infrastructure in one go. It's your team's API, though, and if this is the UX you want we'll accept it. I will note, our team *will not* amend this ourselves if that's the chosen path.

Reply:
This is the known limitation for now that DDL statements are not part of create database call.
cc: @aseering